inc/common.class.php

```diff
@@ -884,33 +767,7 @@ public static function getAvailableCategories($helpdeskHome = 1): DBmysqlIterato
     * @return array
     */
    public static function getAvailableCategoriesCriterias($helpdeskHome = 1): array {
```

Comment:
Code was identical so I've called the function from `PluginFormcreatorCommon` directly

Reply:
I would do the reverse (like for getCategoryTree) as this is related to categories.

Reply:
Indeed this was a mistake, fixed in ecba050d.